### PR TITLE
Handle `planetariumhq/lib9c-stateservice` too

### PR DESCRIPTION
**Pull request by moreal**

### CODE REVIEW

File: toolbelt/utils/converter.py [hunk 1]
```diff
@@ -13,6 +13,7 @@
 def dockerhub2github_repo(dockerhub_repo: str):
     dockerhub2github_repo_map = {
         "ninechronicles-headless": HEADLESS_REPO,
+        "lib9c-stateservice": HEADLESS_REPO,
```
Comment (Atralupus) [inline toolbelt/utils/converter.py:16]: It's not independent service?

Comment (moreal) [reply toolbelt/utils/converter.py:16]: It is an independent service but it uses the same repository now.

Comment (Atralupus) [reply toolbelt/utils/converter.py:16]: Ok, good

**Review by Atralupus — APPROVED** [review]

**Comment by moreal** [comment]
Because I don't have enough permission to merge this pull request, so could you merge this pull request? @Atralupus 
The `Check / pyscripts-test` job seems failed in the main branch, too.

**Pull request merged**